platform/platform-configuration/src/test/java/org/codice/ddf/configuration/PlatformUiConfigurationTest.java

```diff
@@ -64,7 +67,10 @@ public void testConfig() throws IOException {
         assertEquals("footer", jsonObject.get(PlatformUiConfiguration.FOOTER));
         assertEquals("background", jsonObject.get(PlatformUiConfiguration.BACKGROUND));
         assertEquals("color", jsonObject.get(PlatformUiConfiguration.COLOR));
-        assertEquals("product", jsonObject.get(PlatformUiConfiguration.VERSION));
+        assertEquals("product", jsonObject.get(PlatformUiConfiguration.TITLE));
+        //TODO Tarr Identify if this is a valid check (should "version" be considered the number or the product name.
+        //TODO (cont'd) Most other places refer to the version number as "version" and the product name, eg "DDF" as "product name" or "title"
+        //assertEquals("product", jsonObject.get(PlatformUiConfiguration.VERSION));
```

Comment:
@stustison I believe you are the person who introduced this assertion. Should the `PlatformUiConfiguration.VERSION` be something like "product" or "1.0.0" or "product 1.0.0"?